infullmvp-kotlin-koin/src/main/java/com/infullmobile/android/infullmvp/InFullMvpActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.infullmobile.android.infullmvp
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.support.annotation.CallSuper
+import android.support.v7.app.AppCompatActivity
+import org.koin.android.scope.ext.android.bindScope
+import org.koin.android.scope.ext.android.getOrCreateScope
+
+abstract class InFullMvpActivity<out PresenterType : InFullMvpPresenter<*>>
+    : AppCompatActivity(), MvpView {
```

Comment:
Android components are not implementing views.
This is the key decision for the whole library. It should not be changed.

Reply:
I've changed that.
It wasn't easy, but it's working!
We can use Synthetic accessors inside View!